semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -261,6 +255,57 @@ public static boolean isSqlTypeBackendForOtelFormat(
     return Optional.empty();
   }
 
+  public static Optional<String> getCassandraURI(Event event) {
+    if (SpanAttributeUtils.containsAttributeKey(event, OTEL_DB_CONNECTION_STRING)) {
+      String uri =
+          StringUtils.substringAfter(
+              StringUtils.trim(
+                  SpanAttributeUtils.getStringAttribute(event, OTEL_DB_CONNECTION_STRING)),
+              ":");
+      return getHostPortFromURI(uri);
+    }
+    Optional<String> OtBackendURI = getBackendURIForOpenTracingFormat(event);
+    if (OtBackendURI.isPresent()) {
```

Comment:
Same comment as `getElasticsearchURI` about inconsistency with corresponding method to check backend type